contrib/kubernetes/manifests/registry-gcs-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcs-keyfile
+type: Opaque
+data:
+  key.json: {base64 encode of the service account json file data}
```

Comment:
I think it would be clearer to say:
```yaml
data:
  key.json: {base64-encoded service account JSON file data}
  bucket: {base64-encoded bucket name}
```